core/templates/components/shared-component.module.ts

```diff
@@ -84,7 +88,7 @@ import { TakeBreakModalComponent } from
     SharingLinksComponent,
     SkillMasteryViewerComponent, AttributionGuideComponent,
     LazyLoadingComponent, LoadingDotsComponent, SocialButtonsComponent,
-    ProfileLinkImageComponent, ProfileLinkTextComponent,
+    PromoBarComponent, ProfileLinkImageComponent, ProfileLinkTextComponent,
```

Comment:
Can you convert them into one line... i.e. one component per line. Also, I think you may have to put this value in exports array as well.

Reply:
done!